### DISCUSSION TITLE
First cell should receive focus when a Notebook is created

### DISCUSSION THREAD

**Discussion by burkeholland** [microsoft/vscode-jupyter#9225] [New Ideas]
The default behaviour in VS Code for new file is to set the focus to the editor after the file is created. For Notebooks, I expect that when the notebook is created, the focus will be in the first cell that is automatically added to the notebook. The current behavior is that the notebook gets the focus, but its 4 tab stops to get the cursor into the actual cell.

https://user-images.githubusercontent.com/686963/156426164-151dc04b-0f25-4eb7-a37f-95d1a5d8b061.mp4

**Comment by rebornix** [comment]
This is currently intentional and tuned for keyboard users (or Jupyter users in overall). Once a notebook is created, users can press enter to edit the cell content, or press `a/b` to insert cells. The behavior is the same as Jupyter Lab as well.
